assets/locales/en.json

```diff
@@ -32,22 +32,78 @@
       "description": "Dive into our latest blog posts to learn more about the entire project and our collaboration with OceanDAO.",
       "title": "Blog"
     },
-    "title": ["Blog", "Posts"]
+    "title": [
+      "Blog",
+      "Posts"
+    ]
   },
   "creator": {
     "overview": "Overview",
     "progress": "Progress: {progress}%",
     "project": {
+      "basic": {
+        "logo": {
+          "label": "Add your wonderful project logo"
+        },
+        "oneLiner": {
+          "label": "Your project in one sentence",
+          "placeholder": "Write a short description of your project..."
```

Comment:
suggestion(non-blocking): i think that the ... ending implies that the sentence has more info when you hover over it, can't we just use . or no marks at all?